processor/transform/readme-components.ts

```diff
@@ -89,12 +90,12 @@ const coerceJsxToMd =
         ...(width && { width }),
         ...(height && { height }),
         alt,
-        children: caption ? mdast(caption).children : (node.children as any),
         title,
       };
 
       const mdNode: ImageBlock = {
         ...attrs,
+        children: caption ? mdast(caption).children : node.children,
```

Comment:
This was incorrect before and should not have been copying `children` onto the attributes.